### PR TITLE
BREAKING CHANGE: removed exceptions made to indent

### PR DESCRIPTION
**Pull request by ritazoliveira**
Removed the `SwitchCase` and `MemberExpression` indent rule exceptions .

### CODE REVIEW

**Review by acostalima — CHANGES_REQUESTED** [review]
We should add a few more samples to demonstrate how function calls can be chained without errors.

**Review by acostalima — APPROVED** [review]
LGTM! 👍

**Comment by acostalima** [comment]
We should squash into a single commit and [add "BREAKING CHANGE" to the commit message body](https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer)!

File: test/__snapshots__/index.spec.js.snap [hunk 1]
```diff
@@ -1297,6 +1309,12 @@ Array [
     "rule": "array-bracket-spacing",
     "severity": 2,
   },
+  Object {
```
Comment (satazor) [inline test/__snapshots__/index.spec.js.snap:1300]: This seems to be a rule related to `block-spacing` and not indent. Check `style.js#18`

File: test/__snapshots__/index.spec.js.snap [hunk 2]
```diff
@@ -1309,6 +1327,12 @@ Array [
     "rule": "block-spacing",
     "severity": 2,
   },
+  Object {
```
Comment (satazor) [inline test/__snapshots__/index.spec.js.snap]: Same.

**Review by satazor — APPROVED** [review]

**Pull request merged**